src/core/turn-order.js

```diff
@@ -78,4 +88,23 @@ export const TurnOrder = {
       }
     },
   },
+
+  CUSTOM: {
+    //  order: (G, ctx) => {
+    // -      if (G === undefined || G.playOrder === undefined) {
+    // -        let playOrder = [];
+    // -        for (let i = 0; i < ctx.numPlayers; i++) {
+    // -          playOrder.push(i);
+    // -        }
+    // -        G = { ...G, playOrder };
+    // -      }
+    // -      return G;
+    first: G => G.playerOrder[0] + '',
+    next: (G, ctx) => {
+      let playerOrderPos = G.playerOrder.findIndex(
+        nextPlayer => nextPlayer == ctx.currentPlayer
```

Comment:
why not strict equal??

Reply:
currentPlayer is a string (usually), passOrder and playerOrder a array of integer